src/forms/dispense-form.component.tsx

```diff
@@ -5,33 +5,30 @@ import {
   showNotification,
   useLayoutType,
 } from "@openmrs/esm-framework";
-import { Button, FormLabel, DataTableSkeleton } from "@carbon/react";
+import { Button, FormLabel } from "@carbon/react";
 import styles from "./dispense-form.scss";
 import { closeOverlay } from "../hooks/useOverlay";
 import { MedicationDispense } from "../types";
 import { saveMedicationDispense } from "../medication-dispense/medication-dispense.resource";
-import MedicationDispenseReview from "../components/medication-dispense-review.component";
+import MedicationDispenseReview from "./medication-dispense-review.component";
 
 interface DispenseFormProps {
-  medicationDispenses: Array<MedicationDispense>;
+  medicationDispense: MedicationDispense;
```

Comment:
Basically, the majority of these changes are changing various components that formerly had an array of medication dispenses as a prop to only take a single medication dispense, since we moving the Dispense button to the request, not encounter, level.  This simplifies things. (I hope!)